dynamic_programming/trapped_rainwater.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ * @brief Implementation of the Trapped Rainwater Problem
```

Comment:
Maybe include a link to a reliable source explaining the algorithm/implementation, as suggested [here](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program).

Reply:
Yeah, I will add links to the algorithm explanation to the documentation.